_data/simple-icons.json

```diff
@@ -16158,6 +16158,12 @@
             "source": "https://tripadvisor.mediaroom.com/logo-guidelines",
             "guidelines": "https://tripadvisor.mediaroom.com/logo-guidelines"
         },
+        {
+            "title": "trivago",
+            "hex": "E32851",
+            "source": "https://cloud.brandmaster.com/point/en/trivagobrandhub",
+            "guidelines": "https://cloud.brandmaster.com/point/en/trivagobrandhub"
```

Comment:
Hmm not sure what our policy on it but since https://company.trivago.com/press-media/ is on the official website and links to the ones you have listed here, might be better to keep it clear that they are official sources.

Reply:
@uncenter thanks for your url, updated just now